### PR TITLE
Added Instrumentation Test and Unit Tests

### PR DESCRIPTION
**Pull request by Smarshal21**
**Added Instrumentation Test For VibrationUtils and Unit Tests for other files mentioned below.**

- [x] VibrationUtils
- [x] BundleKeys
- [x] UriUtils
- [x] DataChannelMessageNotifier
- [x] Globals
- [x] PeerConnectionNotifier

### CODE REVIEW

File: app/src/androidTest/java/com/nextcloud/talk/utils/VibrationUtilsTest.kt [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/*
+ * Nextcloud Talk application
+ *
+ * @author Mario Danic
+ * Copyright (C) 2017-2019 Mario Danic <mario@lovelyhq.com>
```
Comment (mahibi) [inline app/src/androidTest/java/com/nextcloud/talk/utils/VibrationUtilsTest.kt:5]: please change the author in the copyright headers

Comment (Smarshal21) [reply app/src/androidTest/java/com/nextcloud/talk/utils/VibrationUtilsTest.kt:5]: Oh then can i write my name?

**Comment by mahibi** [comment]
lgtm, thanks!

just as a reminder now that you use the original repo: to update the branch you can rebase on master instead to merge.

File: app/src/androidTest/java/com/nextcloud/talk/utils/VibrationUtilsTest.kt (hunk 1 above)
Comment (Smarshal21) [reply app/src/androidTest/java/com/nextcloud/talk/utils/VibrationUtilsTest.kt]: Done

**Comment by mahibi** [comment]
@Smarshal21 if you want to have all checks green:

- "Check / check (ktlintCheck) (pull_request)"  
--> click on details which will show:
```

 > Task :app:ktlintTestSourceSetCheck FAILED
/home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:60:5 Declarations and declarations with annotations should have an empty space between.
/home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:83:22 Argument should be on a separate line (unless all arguments can fit a single line)
/home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:83:22 Missing newline after "("
/home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:84:67 Missing newline before ")"
/home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:84:68 Missing newline before ")"
/home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:85:22 Argument should be on a separate line (unless all arguments can fit a single line)
/home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:85:22 Missing newline after "("
/home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:86:67 Missing newline before ")"
/home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:86:68 Missing newline before ")"
/home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/UriUtilsTest.kt:29:1 First line in a method block should not be empty
/home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/webrtc/PeerConnectionNotifierTest.kt:34:5 Declarations and declarations with annotations should have an empty space between.
```


- "DCO Action required after 1s — DCO"  
--> see https://github.com/nextcloud/talk-android/blob/master/CONTRIBUTING.md#sign-your-work


feel free to fix or ask if anything is unclear.

**Comment by Smarshal21** [comment]
> @Smarshal21 if you want to have all checks green:
> 
> * "Check / check (ktlintCheck) (pull_request)"
>   --> click on details which will show:
> 
> ```
> 
>  > Task :app:ktlintTestSourceSetCheck FAILED
> /home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:60:5 Declarations and declarations with annotations should have an empty space between.
> /home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:83:22 Argument should be on a separate line (unless all arguments can fit a single line)
> /home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:83:22 Missing newline after "("
> /home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:84:67 Missing newline before ")"
> /home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:84:68 Missing newline before ")"
> /home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:85:22 Argument should be on a separate line (unless all arguments can fit a single line)
> /home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:85:22 Missing newline after "("
> /home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:86:67 Missing newline before ")"
> /home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/BundleKeysTest.kt:86:68 Missing newline before ")"
> /home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/utils/UriUtilsTest.kt:29:1 First line in a method block should not be empty
> /home/runner/work/talk-android/talk-android/app/src/test/java/com/nextcloud/talk/webrtc/PeerConnectionNotifierTest.kt:34:5 Declarations and declarations with annotations should have an empty space between.
> ```
> 
> * "DCO Action required after 1s — DCO"
>   --> see https://github.com/nextcloud/talk-android/blob/master/CONTRIBUTING.md#sign-your-work
> 
> feel free to fix or ask if anything is unclear.

Sure

**Pull request merged**